libs/common/src/sql-db-connection/migrations/1732803141766-AddFeeRecordCorrectionTable.ts

```diff
@@ -12,15 +12,15 @@ export class AddFeeRecordCorrectionTable1732803141766 implements MigrationInterf
                 "dateRequested" datetime2 NOT NULL CONSTRAINT "DF_489233b45b51a528c7c12d9e999" DEFAULT getdate(),
                 "isCompleted" bit NOT NULL,
                 "feeRecordId" int NOT NULL,
+                "bankTeamName" nvarchar(500),
+                "bankTeamEmails" nvarchar(1000),
```

Comment:
Sorry I didn't notice this before but we should never edit a migration that has already been merged in as it will have been run on some environments e.g. dev already and possibly even higher envs depending on when release branches get cut. Running migrations will just run the ones that have been added since the last migration run and editing old ones might even throw errors.

Please can you remove these lines and then follow the instructions for making a new migration in `doc/sql-db.md`.

Reply:
done